scenario/mocking.py

```diff
@@ -540,7 +540,7 @@ def action_get(self):
     def storage_add(self, name: str, count: int = 1):
         if not isinstance(count, int) or isinstance(count, bool):
             raise TypeError(
-                f"storage count must be integer, got: {count} ({type(count)})",
+                f"storage count must be integer, got: {count} ({type(count)}",
```

Comment:
Maybe you meant this?

```suggestion
                f"storage count must be integer, got: {count} {type(count)}",
```

Or was the change not intended? 🤔

Reply:
That's [this super annoying bug](https://matrix.to/#/!eNCNzcteYUDDYpStsu:ubuntu.com/$PWvNehJR9JbDvANs97NkNtIc7WXnXUXrIa9rYA509tg?via=ubuntu.com&via=matrix.org). It's fixed now in main, I'll pull that in and fix this branch too.